packages/pytest-simcore/src/pytest_simcore/helpers/typing_env.py

```diff
@@ -0,0 +1,3 @@
+from typing import Dict, Union
+
+EnvVarsDict = Dict[str, Union[str, None]]
```

Comment:
```suggestion
EnvVarsDict = Dict[str, Optional[str]]
```

Reply:
Yes, but since they mean the same, i prefer to stress here that ``None`` is a possible value which btw it was not obvious at me at first that you could create env files as
```envfile
NAME=VALUE
ONLY_NAME=
```

Reply:
#2760

Reply:
ok i changed it.